src/views/Home/components/About/AboutBox/AboutBox.scss

```diff
@@ -4,7 +4,8 @@
 .about-box {
     z-index: 1;
     border-radius: 33px;
-    width: 80%;
+    width: 70%;
+    max-width: 1500px;
     height: 100%;
     background-color: var(--color-dark-navy-blue);
     margin: 50px 0 50px 0;
```

Comment:
Could we also set a min-height to `&__title` below? For example, something like:

```css
        display: flex;
        flex-direction: column;
        justify-content: center;
        min-height: 50px;
```

Reply:
Yeah I can add that as well, let me test and see how that'll look on smaller screens